rsconf/component/rsconf.py

```diff
@@ -12,8 +12,9 @@
 from rsconf import component
 from urllib.parse import urlparse
 
-_PASSWD_SECRET_JSON_F = "rsconf_auth.json"
 PASSWD_SECRET_F = "rsconf_auth"
+_KICKSTART_ROOT_D = "/srv/kickstart"
```

Comment:
/srv shouldn't be hardwired. Use host_run_d.

Reply:
170748e